CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 0.5.0
+
+This is a feature release that also contains bug fixes.
+
+* [TK-404](https://tickets.puppetlabs.com/browse/TK-404) - Added a new "v2" API that is based on the [Jolokia](https://jolokia.org) library. Jolokia
+  provides a superset of functionality found in the current "v1" metrics API. Refer to 'document/configureation.md'
+  for more information on setting up Jolokia with Trapperkeeper. The [Jolokia Documentation](https://jolokia.org/documentation.html)
```

Comment:
This subsection of the docs might be better for folks who want to dive right into making requests to the API: https://jolokia.org/reference/html/protocol.html